pages/common/jest.md

```diff
@@ -7,15 +7,19 @@
 
 `jest`
 
-- Run only tests with a name that matches the regex pattern:
+- Run tests from the test files that match the regex patterns:
+
+`jest {{test_file1}} {{test_file2}}`
+
+- Run tests whose names match the regex pattern:
 
 `jest -t {{spec_name}}`
 
-- Run only tests relating to uncommitted files:
+- Run tests related to uncommitted files:
 
-`jest --onlyChanged`
+`jest -o`
```

Comment:
I think it seems better to keep this as the long option as there isn't anything in the new description that matches the `o`. Or update the description to mention it `[O]nly run tests related to uncommitted files:` as before. 😕

Reply:
Yep maybe it's better to keep longer flags.

Reply:
I don't mind either way, but I think it should be consistent. The long name for `-t` is `--testNamePattern`, which I think is too long. Can I have the short name first, then the long name below it?

So:

```
- Run tests whose names match the regex pattern:

`jest -t {{spec_name}}`
`jest --testNamePattern {{spec_name}}`
```

Reply:
I think it's fine to just use the long name as users can always run `--help` to find the short name? We don't currently support multiple versions of a flag in the spec afaik.

Reply:
Sure, just the long name is fine, then.